src/app/common/templates/config-data.yaml

```diff
@@ -36,6 +36,7 @@ CommandHandlerInterfaceOnlyClusters:
     - Microwave Oven Control
     - Energy EVSE
     - Energy EVSE Mode
+    - Energy Calendar
```

Comment:
This is a claim about implementation.  It does not belong in this PR.

Reply:
Fix